test/debezium-avro/README.md

```diff
@@ -0,0 +1,9 @@
+This is an end-to-end test that spawns a Postgres -> Debezium -> Kafka -> Materialize pipeline
+and then performs various operations on it, including DDL and schema migrations
+
+To run:
+
+./mzcompose down -v ; ./mzcompose run debezium-avro
+
+The tests are numbered so that 01-initialize.td runs before all others and 9*.td run
+after all others, since they modify the Debezium configuration on the fly.
```

Comment:
It would be nice to figure out how to share setup / teardown code across these test cases so that we don't need to run all of the tests in enumeration order.

For example, I frequently run `TD_TEST=abc.td ./mzcompose run testdrive` to limit the number of test cases that I need to run.

I wanted to make a note of it and I don't think you need to change anything in this PR.

Reply:
I will add the initialization as a separate step in the workflow and code in support for ```TD_TEST``` . Thanks!